pkg/get/tools.go

```diff
@@ -1134,5 +1134,35 @@ https://github.com/{{.Owner}}/{{.Repo}}/releases/download/{{.Version}}/{{.Name}}
 				https://github.com/{{.Owner}}/{{.Repo}}/releases/download/v{{.Version}}/{{.Name}}_{{.Version}}_{{$osString}}_{{$arch}}{{$ext}}`,
 		})
 
+	tools = append(tools,
+		Tool{
+			Owner:   "FairwindsOps",
+			Repo:    "polaris",
+			Name:    "polaris",
+			Version: "3.2.1",
+			URLTemplate: `
```

Comment:
From the looks of this, we can use BinaryTemplate which is easier to maintain, since it only needs to build the binary download name (final part of the URL)

Please could you try converting it?

Reply:
Ok, did not know. I changed it :)